app/src/main/java/com/techcourse/AppWebApplicationInitializer.java

```diff
@@ -15,9 +15,7 @@ public class AppWebApplicationInitializer implements WebApplicationInitializer {
     @Override
     public void onStartup(final ServletContext servletContext) {
         final var dispatcherServlet = new DispatcherServlet();
-        dispatcherServlet.addHandlerMapping(new ManualHandlerMapping());
-        dispatcherServlet.addHandlerAdapter(new ManualHandlerAdapter());
-        dispatcherServlet.addHandlerMapping(new AnnotationHandlerMapping("com"));
+        dispatcherServlet.addHandlerMapping(new AnnotationHandlerMapping(getClass().getPackageName()));
```

Comment:
```suggestion
        dispatcherServlet.addHandlerMapping();
```
해당 코드는 인자를 주지 않아도 정상적으로 실행되더라고요!

Reply:
이부분도 수정했습니다!